pages/search/advanced.tsx

```diff
@@ -157,108 +156,75 @@ export default function AdvancedSearch({ isAuthenticated }) {
           action={`${BASE_URL}/search`}
           onSubmit={handleSubmit}
         >
-          <FormField
+          <Flex
```

Comment:
We briefly talked about this. I thought you were not going to use this only for the button group. Why the need to replace it entirely?

Reply:
The mobile view is now lost (compare it against prod):
<img width="472" alt="Screenshot 2024-10-25 at 4 18 55 PM" src="https://github.com/user-attachments/assets/396a5f1e-0a2d-44ee-98c0-5e2fd98a1684">

Reply:
Got a little trigger happy, I guess. I will replace.

Reply:
updated so mobile view is back to columns